docs/1.18.x/setup/configuration/file-structure.md

```diff
@@ -0,0 +1,39 @@
+# Setup - Configuration
+## Folder Naming
+
+The usual convention for folders is `com/yourName/modName`, but of course you don't have to stick to it.
+
+Each modder has their own style of folder naming that is very similar, for each content package, such as your code for blocks, I would suggest either `block` or `blocks`. This goes in your `common` package (read below)
+
+##  Organisation
+
+All your content goes in your `src/main/java/com/yourName/modName` package from your main folder.
+There are usually two packages made in every mod, but you don't have to have them all or name them the same:
+
+- Common
+- Client
```

Comment:
This isn't used by a lot of mods, only some, technically it doesn't matter, I wouldn't go into so much detail over client and common and core, ect...

Instead, you can show a file tree like this:
```
src/main/java/
└─ com.yourname.examplemod/
    └─ Main.java
```

That is enough for now.